src/raftstore/store/region_snapshot.rs

```diff
@@ -38,13 +42,41 @@ impl RegionSnapshot {
         RegionSnapshot {
             snap,
             region: Arc::new(region),
+            // Use 0 to indicate that we the apply index is missing and we need to KvGet it,
```

Comment:
Use 0 to indicate that <del>we</del> the apply index is missing and we need to KvGet it,